index.html

```diff
@@ -64,6 +62,39 @@ <h2>
     <p>
       TBW
     </p>
+    <p id="credential-type-examples">
+      There are many types of digital credential that can be conveyed using
+      this API. Some of these types include:
+    </p>
+    <ul>
+      <li>
+        a driver's license issued by a testing facility
+      </li>
+      <li>
+        a passport issued by a government
+      </li>
+      <li>
+        a student identification card or diploma issued by a university
+      </li>
+      <li>
+        an employee identification card issued by a company
+      </li>
+      <li>
+        a traveler's boarding pass issued by an airline
+      </li>
+      <li>
```

Comment:
This probably falls more on FedCM...

Reply:
Same as comment in: https://github.com/WICG/digital-identities/pull/82/files#r1499358644

Reply:
Reduced the list down to the only credential type we have consensus on right now: 06271a72f5fcfbef716f61d49574b0de9449d5d0. Preserved the full list in https://github.com/WICG/digital-identities/issues/85#issuecomment-1963077832.